### ISSUE TITLE
text-space-collapse

### ISSUE THREAD

**Issue by jensimmons** [Fyrd/caniuse#6350]
There's a page for `text-space-collapse` at https://caniuse.com/sr_text-space-collapse, It does not have a data visualization of browser support. It simply says:

> CSS text-space-collapse property
> This CSS property (formerly known as white-space-collapse or white-space-collapsing) is not supported  in any modern browser, nor are there any known plans to support it.

This feels out of date to me. `text-space-collapse` is a real CSS property that is officially part of [Text level 4](https://www.w3.org/TR/css-text-4/#white-space-collapsing). It has not been rejected or abandoned. No browser yet supports `text-space-collapse`, but the current page on Can I Use makes it seem like this property was an old idea that should be left in the past. 

I want to update the page, adding the data for building a data visualization and rewriting the description to match others (adding information about what it does and removing the editorial judgement about whether browsers plan to implement in the future). But I can't find the page in the code repo. What does the `sr_` mean in the URL? Where does the JSON file for this property live?

**Comment by Schweinepriester** [comment]
> But I can't find the page in the code repo. What does the `sr_` mean in the URL? Where does the JSON file for this property live?

To give you my best answer until he responds: Afaik `sr` stands for search result, as in there is no support table but only this search result (another example: https://caniuse.com/sr_jquery). To my knowledge those search results sadly aren't open source (yet?).

Initial decision was here: https://github.com/Fyrd/caniuse/issues/992#issuecomment-227650879

> […] so have added it as a basic search results entry until that changes

I guess the time is coming :)
Are there bugs from any vendor?

**Comment by Fyrd** [comment]
Yes, the search result entries are more of a site feature for convenience to save people some time looking things up without adding unnecessary bloat to the data set.

This particular feature has been in the spec for many years but no browser has appeared to show an interest in implementing it. Stating that there are no known plans to support it seems accurate to me until there's evidence to the contrary. 

As @Schweinepriester mentioned I'll of course be happy to remove/update it once any browser appears to show an interest in implementing it. Sounds like we might be able to expect this soon?
